### ISSUE TITLE
Unable to delete Collection folder in GUI in case folder does not exist on filesystem

### ISSUE THREAD

**Issue by martinsefcik** [usebruno/bruno#314]
When folder is deleted on filesystem (outside Bruno) then it stays in Bruno and it is not possible to delete it due to the following error message:

```
Uncaught (in promise) Error: Error invoking remote method 'renderer:delete-item': Error: The directory does not exist
```

I guess this error should be ignored.

**Comment by martinsefcik** [comment]
@lared I can reproduce it at least with versions 0.18.0, 0.17.0, 0.16.6 on Windows 11.

**Comment by martinsefcik** [comment]
Steps to reproduce:
1. Create new subfolder in collection via some file commander (outside Bruno)
2. Folder is immediately showed Bruno UI
3. Delete this folder via file commander
4. Folder is still visible in Bruno UI
5. Try to Delete folder via Bruno UI -> nothing happen - no error popup, folder is not deleted, and the error message mention here in before is printed in Developer console.

**Comment by martinsefcik** [comment]
I cannot anymore reproduce it. Probably fixed in #465
